dungeons/microdungeons/randomencounter/unsorted/generic/randomencounter.dungeon.patch

```diff
@@ -0,0 +1,56 @@
+[
```

Comment:
Ack! Ya I forgot the double spaces on that one for generic/unsorted, I know
you'll scold me lol...it may end up getting deleted anyway as I was digging
into the assets and can't really seem to find where vanilla
"randomencounter.dungeon" is generated anymore, all I was seeing is biome
specific microdungeons and miniboss ones.

Maybe I'll check pleased/upbeat to see if it's any different. If anything,
I'll add the old bandit spawns to a seperate mod I'm working on and make a
compatibility patch for it...and we can just delete the extra lines out of
the terrestrial patch to keep it simple for all the people lol.

At work right now will edit those items discussed.

-L
On Oct 19, 2015 2:23 AM, "Fevix" <notifications@github.com> wrote:

> In
> dungeons/microdungeons/randomencounter/unsorted/generic/randomencounter.dungeon.patch
> <https://github.com/Fevix/Avali/pull/79#discussion_r42339012>:
>
> > @@ -0,0 +1,56 @@
> > +[
>
> Please use 2x spaces for indentation. Consistency and all.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Fevix/Avali/pull/79/files#r42339012>.
>